src/main/scala/centaur/test/workflow/WorkflowData.scala

```diff
@@ -1,32 +1,33 @@
 package centaur.test.workflow
 
 import java.nio.file.Path
-import java.nio.file.Files
 
 import better.files.File
 import cats.data.Validated._
 import centaur.test._
 import com.typesafe.config.Config
-import configs.Result
+import configs.Result.{Failure, Success}
 import configs.syntax._
-import spray.json.JsArray
+import cromwell.api.model.Label
+import spray.json._
 
-case class WorkflowData(wdl: String, inputs: Option[String], options: Option[String], zippedImports: Option[File])
+
+case class WorkflowData(wdl: String, inputs: Option[String], options: Option[String], labels: Option[List[Label]], zippedImports: Option[File])
```

Comment:
I would say it's fine to just use `List[Label]` since that can already be empty.